### DISCUSSION TITLE
Github Copilot included with Github Pro

### DISCUSSION THREAD

**Discussion by phocodes** [community/community#19070] [Copilot Conversations] [answered]
Github Copilot should be included with Github Pro or Github Student
Either make a new Pro Plan or add a downgraded version of Github Copilot.
But I highly doubt people would pay $10(USD) for a AI code completion tool.

**Comment by ldeluigi** [comment, marked as answer]
It's included in the student developer package

**Reply by warengonzaga** [reply]
@ldeluigi what about GitHub Pro?

**Reply by ldeluigi** [reply]
idk, I think it's and add-on
